docs/index.bs

```diff
@@ -412,77 +400,75 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
     </section>
 
     <section algorithm="navigator-service-worker-waiting">
       <h4 id="navigator-service-worker-waiting">{{ServiceWorkerRegistration/waiting}}</h4>
 
-      <p><dfn attribute for="ServiceWorkerRegistration"><code>waiting</code></dfn> attribute <em class="rfc2119" title="MUST">must</em> return the value to which it was last set.</p>
+      <dfn attribute for="ServiceWorkerRegistration"><code>waiting</code></dfn> attribute *must* return the value to which it was last set.
 
-      <p class="note">The {{ServiceWorker}} objects returned from this attribute getter that represent the same <a href="#dfn-service-worker">service worker</a> are the same objects.</p>
+      Note: The {{ServiceWorker}} objects returned from this attribute getter that represent the same <a for="/">service worker</a> are the same objects.
     </section>
 
     <section algorithm="navigator-service-worker-active">
       <h4 id="navigator-service-worker-active">{{ServiceWorkerRegistration/active}}</h4>
 
-      <p><dfn attribute for="ServiceWorkerRegistration"><code>active</code></dfn> attribute <em class="rfc2119" title="MUST">must</em> return the value to which it was last set.</p>
+      <dfn attribute for="ServiceWorkerRegistration"><code>active</code></dfn> attribute *must* return the value to which it was last set.
 
-      <p class="note">The {{ServiceWorker}} objects returned from this attribute getter that represent the same <a href="#dfn-service-worker">service worker</a> are the same objects.</p>
+      Note: The {{ServiceWorker}} objects returned from this attribute getter that represent the same <a for="/">service worker</a> are the same objects.
     </section>
 
     <section algorithm="service-worker-registration-scope">
       <h4 id="service-worker-registration-scope">{{ServiceWorkerRegistration/scope}}</h4>
 
-      <p>The <dfn attribute for="ServiceWorkerRegistration"><code>scope</code></dfn> attribute <em class="rfc2119" title="MUST">must</em> return <a for="ServiceWorkerRegistration">service worker registration</a>'s <a lt="URL serializer">serialized</a> <a href="#dfn-scope-url">scope url</a>.</p>
+      The <dfn attribute for="ServiceWorkerRegistration"><code>scope</code></dfn> attribute *must* return <a for="ServiceWorkerRegistration">service worker registration</a>'s <a lt="URL serializer">serialized</a> <a for="service worker registration">scope url</a>.
 
-      <p class="example">In the example in section 3.1.1, the value of <code>registration.scope</code>, obtained from <code>navigator.serviceWorker.ready.then(function(registration) { console.log(registration.scope); })</code> for example, will be "<code>https://example.com/</code>".</p>
+      <div class="example">
+        In the example in section 3.1.1, the value of <code>registration.scope</code>, obtained from <code>navigator.serviceWorker.ready.then(registration => console.log(registration.scope))</code> for example, will be "<code>https://example.com/</code>".
```

Comment:
Maybe change `section 3.1.1` to `[[#service-worker-url]]` to get auto-linking to that section? Although not sure if having the section title in the resulting link will make it more or less readable here.